app/scripts/background.js

```diff
@@ -892,6 +891,9 @@ export function setupController(
       controller.isClientOpen = true;
       controller.setupTrustedCommunication(portStream, remotePort.sender);
 
+      // initialize the request to fetch remote feature flags
```

Comment:
Nit: I'd recommend moving this comment to a JSDoc comment above the `initializeRemoteFeatureFlags` function itself.

We haven't enabled the rule yet here, but part of our standard linting rules is to require a JSDoc block above every function to explain what it's for.

Reply:
That's a good pattern to follow, addressed 2035e07f4fdb4bf02b500e4935f2f27fe278b857